xarray/core/ops.py

```diff
@@ -520,24 +522,40 @@ def inject_bottleneck_rolling_methods(cls):
     for name, f in methods:
         func = cls._reduce_method(f)
         func.__name__ = name
-        func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(name=func.__name__)
+        func.__doc__ = _ROLLING_REDUCE_DOCSTRING_TEMPLATE.format(
+            name=func.__name__)
         setattr(cls, name, func)
 
     # bottleneck rolling methods
     if has_bottleneck:
-        if LooseVersion(bn.__version__) < LooseVersion('1.0'):
+        bn_version = LooseVersion(bn.__version__)
```

Comment:
Let's add a TODO to do that before releasing v0.10. But for now, we should keep it rather than updating the required version in a patch release (supposing we issue a v0.9 release).